src/steps/users/index.ts

```diff
@@ -0,0 +1,52 @@
+import {
+  createDirectRelationship,
+  Entity,
+  IntegrationStep,
+  IntegrationStepExecutionContext,
+  RelationshipClass,
+} from '@jupiterone/integration-sdk-core';
+
+import { createAPIClient } from '../../client';
+import { IntegrationConfig } from '../../config';
+import {
+  ACCOUNT_ENTITY_KEY,
+  Entities,
+  Relationships,
+  Steps,
+} from '../constants';
+import { createUserEntity } from './converter';
+
+export async function fetchUsers({
+  instance,
+  jobState,
+}: IntegrationStepExecutionContext<IntegrationConfig>) {
+  const apiClient = createAPIClient(instance.config);
+  const accountEntity = await jobState.getData(ACCOUNT_ENTITY_KEY);
+
+  await apiClient.iterateUsers(async (user) => {
+    const userEntity = createUserEntity(user);
+
+    if (userEntity) {
+      await jobState.addEntity(userEntity);
+      if (accountEntity)
```

Comment:
Same as above. Also, can accountEntity ever be nullable? If not, we should remove `if (accountEntity) {}` check.

Reply:
The `accountEntity` is coming from `jobState.getData()`, I agree that since we're building it off from env vars it should always exist but just in case something goes wrong in `fetchAccount()` step function I guess having the if check can't hurt :thinking:

Reply:
Hmm, I just removed it since making the `hexnode_account` does not involve any endpoints so failures are highly unlikely. Should I bring it back?